### PR TITLE
Update 2022 ActiveDirectory

### PR DESCRIPTION
**Pull request by PatAltimore**

### CODE REVIEW

**Review by JohanFreelancer9 — APPROVED** [review]

File: docset/winserver2022-ps/activedirectory/Add-ADGroupMember.md [hunk 1]
```diff
@@ -146,6 +146,25 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -DisablePermissiveModify
+Group membership updates use permissive modify by default. This suppresses an error when adding a member that is already member of the group.
+When this parameter is used, an error "The specified account name is already a member of the group" is returned.
+
+This parameter is available in Windows Server 2019 with the September 2020 Updates.
```
Comment (dariomws) [inline docset/winserver2022-ps/activedirectory/Add-ADGroupMember.md:153]:
```suggestion
```

Comment (dariomws) [reply docset/winserver2022-ps/activedirectory/Add-ADGroupMember.md:153]: Maybe this sentence should be removed.

**Review by dariomws — APPROVED** [review]
Just a small detail.

File: docset/winserver2022-ps/activedirectory/Add-ADGroupMember.md (hunk 1 above)
Comment (PatAltimore) [reply docset/winserver2022-ps/activedirectory/Add-ADGroupMember.md:153]: Hi @dariomws, Thanks for the feedback. I noticed the sentence was in Remove-ADGroupMember as well. I removed the sentence in both articles.

Thanks,
Pat

**Review by tseward — APPROVED** [review]

**Comment by PatAltimore** [comment]
#sign-off

**Comment by traya1** [comment]
Applying sign-off

**Comment by v-dihans** [comment]
close/reopen to effect build

**Pull request merged**